site/en/docs/privacy-sandbox/attribution-reporting/prevent-duplication/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Prevent duplication in reports'
+subhead: >
+  Learn how to prevent duplicates in both event-level and aggregatable reports.
+description: >
+  Learn how to prevent duplicates in both event-level and aggregatable reports.
+date: 2023-03-06
+updated: 2023-07-21
+authors:
+  - maudn
+---
+
```

Comment:
I'd recommend adding an intro sentence before going straight into the subheadings. I believe the style guide discourages stacked headings too, but more importantly, I find it helpful when there's a sentence or two telling me what to expect from the content of the page.

Reply:
Added